kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -1,24 +1,31 @@
 #!/bin/bash
 # helper script to deploy given service with given tag to k8s infrastructure
 
-if [ $# -ne 3 ]; then
-    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration>"
+if [ $# -ne 4 ]; then
+    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration> <reference_of_decryption_barbican_secret>"
     exit 1
 fi
 
 ns=$1
 srv=$2
 conf=$3
+secretref=$4
 
     # cmsweb configuration area
     echo "+++ configuration: $conf"
     echo "+++ cms service : $srv"
     echo "+++ namespaces   : $ns"
+    echo "+++ secretref   : $secretref"
 
     if [ ! -d $conf/$srv ]; then
 	echo "Unable to locate $conf/$srv, please provide proper directory structure like <configuration>/<service>/<files>"
   	exit 1
     fi
+   
+    # Get SOPS decryption key and put it in config directory
+    mkdir -p ~/.config/sops/age/
```

Comment:
@Panos512 , unfortunately this is wrong. What if as a user I want to have my own sops/age keys not related to ones we will store to openstack? Therefore without my consent this script will delete my keys. Instead, either use local directory and place there `PWD/.openstack/config/sops/age` area and keys, or rely on common env parameter which should be checked upfront. Moreover, `keys.txt` will be better to replace with `openstack-keys.txt` to distinguish them from `keys.txt` which can be defined and used by end-user.